src/go/k8s/controllers/redpanda/cluster_controller_configuration.go

```diff
@@ -414,7 +414,7 @@ func isSafeToRestart(
 ) bool {
 	configVersions := make(map[int64]bool)
 	for i := range clusterStatus {
-		log.Info(fmt.Sprintf("Node %d is using config version %d", clusterStatus[i].NodeID, clusterStatus[i].ConfigVersion))
+		log.Info(fmt.Sprintf("Node %d is using config version %d", clusterStatus[i].NodeID, clusterStatus[i].ConfigVersion)) // actually pod ordinal
```

Comment:
should all this then say `pod with ordinal X is using....`

Reply:
I think I was wrong. Removed the comment.